.github/workflows/ubuntu.yml

```diff
@@ -84,7 +84,7 @@ jobs:
           then
             echo "BUILD_TYPE=Release" >> $GITHUB_ENV
           else
-            echo "BUILD_TYPE=Debug" >> $GITHUB_ENV
+            echo "BUILD_TYPE=Release" >> $GITHUB_ENV
```

Comment:
I assume this is temporary? Release vs. Release?

Reply:
Yes